image_generation/image_generation.py

```diff
@@ -100,7 +105,7 @@ def preprocess_image(image):
 def generate_from_rhino_view(image, prompt, pipeline=None, negative_prompt="ugly, low quality", 
                              guidance_scale=5, control_strength=0.5, num_inference_steps=8, seed=None):
     
-    pipe = pipeline if pipeline is not None else get_pipeline()
+    pipe = pipeline or get_pipeline() or load_pipeline()
```

Comment:
since we set `_pipeline = None` during flush, we need to load it again once needed